include/envoy/server/access_log_config.h

```diff
@@ -23,10 +23,10 @@ class AccessLogInstanceFactory {
    * Create a particular AccessLog::Instance implementation from a config proto. If the
    * implementation is unable to produce a factory with the provided parameters, it should throw an
    * EnvoyException. The returned pointer should never be nullptr.
-   * @param config the custom configuration for this access log type.
-   * @param filter filter to determine whether a particular request should be logged. If no filter
+   * @param config is the custom configuration for this access log type.
+   * @param filter can determine whether a particular request should be logged. If no filter
```

Comment:
I think the previous state is more doxygen style at least as used in Envoy; it's not a sentence; it's the arg name followed by a description of the arg. For me, you can resolve this by simply reverting this file.